website/src/views/flu.ts

```diff
@@ -0,0 +1,125 @@
+    public readonly dateRangeOptions: DateRangeOption[] = [
+        dateRangeOptionPresets.lastMonth,
+        dateRangeOptionPresets.last2Months,
+        dateRangeOptionPresets.last3Months,
+        dateRangeOptionPresets.last6Months,
+        dateRangeOptionPresets.lastYear,
+        { label: 'Since 2020', dateFrom: '2020-01-01' },
+        { label: '2010-2019', dateFrom: '2010-01-01', dateTo: '2019-12-31' },
+        { label: '2000-2009', dateFrom: '2000-01-01', dateTo: '2009-12-31' },
+        { label: 'Since 2000', dateFrom: '2000-01-01' },
+        { label: 'Before 2000', dateFrom: earliestDate, dateTo: '1999-12-31' },
+        { label: 'All times', dateFrom: this.earliestDate },
+    ];
+    public readonly mainDateField: string;
+    public readonly locationFields: string[];
+    public readonly lineageFilters: LineageFilterConfig[] = [
+        {
+            lapisField: 'subtypeHA',
+            placeholderText: 'HA subtype',
+            filterType: 'text' as const,
+            initialValue: undefined,
+        },
+        {
+            lapisField: 'subtypeNA',
+            placeholderText: 'NA subtype',
+            filterType: 'text' as const,
+            initialValue: undefined,
+        },
+    ];
+    public readonly hostField: string;
+    public readonly authorsField: string | undefined;
+    public readonly authorAffiliationsField: string | undefined;
+    public readonly additionalFilters: Record<string, string> | undefined;
+    public readonly dataOrigins: DataOrigin[] = [dataOrigins.insdc];
+
+    public get additionalSequencingEffortsFields() {
+        return getAuthorRelatedSequencingEffortsFields(this);
+    }
+
+    constructor(organismsConfig: OrganismsConfig) {
+        this.mainDateField = organismsConfig.flu.lapis.mainDateField;
+        this.locationFields = organismsConfig.flu.lapis.locationFields;
+        this.hostField = organismsConfig.flu.lapis.hostField;
+        this.authorsField = organismsConfig.flu.lapis.authorsField;
+        this.authorAffiliationsField = organismsConfig.flu.lapis.authorAffiliationsField;
+        this.additionalFilters = organismsConfig.flu.lapis.additionalFilters;
+    }
+}
+
+export class FluCompareSideBySideView extends BaseView<
+    CompareSideBySideData,
+    FluConstants,
+    GenericCompareSideBySideStateHandler
+> {
+    constructor(organismsConfig: OrganismsConfig) {
+        const constants = new FluConstants(organismsConfig);
+        const defaultPageState = {
+            filters: new Map<Id, DatasetAndVariantData>([
+                [
+                    0,
+                    {
+                        datasetFilter: {
+                            location: {},
+                            dateRange: constants.defaultDateRange,
+                        },
+                        variantFilter: {
+                            lineages: {},
+                            mutations: {},
+                        },
```

Comment:
If we have a reasonable variant that we can put as a default here, I think I'd prefer to add it. Maybe H5? Or H3N2?